src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Traits.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
```

Comment:
This might not be needed. We used it to basically toggle tests that can't run on CI. For example, some tests required the user to pick a contact or the emulator did not have a feature. It may be useful, but it may also really need a new look as this was pretty much hacked as we went along.